model/user-agent/registry.yaml

```diff
@@ -35,3 +35,20 @@ groups:
           using a user-agent for non-browser products, such as microservices with multiple names/versions inside the
           `user_agent.original`, the most significant version SHOULD be selected. In such a scenario it should align
           with `user_agent.name`
+      - id: user_agent.synthetic
```

Comment:
the `user_agent.synthetic` would work great for a boolean flag, but  we have an enum that'll probably grow (we might wat to differentiate `crawler` from `bot` or have a `healthcheck`, `monitoring` in addition to `test`)

So I'm suggesting to find a more precise name.

E.g.  `user_agent.type` that could cover all possible types, synthetic or not - it don't need to be populated for the typical traffic. This might be a bit too broad and people might use it to store something like `mobile` or `browser`- I'm not sure if that's what we want.

Alternative could be `user_agent.synthetic.type` that would only apply to the synthetic requests.

Reply:
I really like the `user_agent.synthetic.type` suggestion. I can see how it might spark confusion to have something like `user_agent.synthetic` since it implies from the naming that this would be boolean. Updating.